### PR TITLE
fix(celery): set celery timezone to UTC

### PR DESCRIPTION
**Pull request by petrjasek**
otherwise it might stop processing tasks for an hour when there is
daylight savings time change.

SDESK-1057

### CODE REVIEW

**Comment by petrjasek** [comment]
@mdhaman when this will be deployed you should remove celerybeatschedule.db file, otherwise you would have ~10h break

**Comment by marwoodandrew** [comment]
Might want to check the celery task that are scheduled to execute once per day at given times! (audit purge, api purge)

**Comment by petrjasek** [comment]
@marwoodandrew good point, made a helper for this so it will use definition in local time and configure it with utc.
also seems like you don't have to remove that schedule db file, it finds out that there was a config change and resets db automatically.

File: superdesk/default_settings.py [hunk 1]
```diff
@@ -55,6 +56,24 @@ def celery_queue(name):
...
+
+if not DEFAULT_TIMEZONE:
+    raise ValueError("DEFAULT_TIMEZONE is empty")
+
+
+def local_to_utc_hour(hour):
+    now = datetime.now()
+    tz = pytz.timezone(DEFAULT_TIMEZONE)
+    local = tz.localize(now.replace(hour=hour))
+    print('local', hour, 'utc', local.utctimetuple()[3])
```
Comment (naspeh) [inline superdesk/default_settings.py]: should be removed :)

Comment (petrjasek) [inline superdesk/default_settings.py]: ups

**Pull request merged**